src/components/ArticlePreview.jsx

```diff
@@ -6,26 +6,37 @@ export default class ArticlePreview extends React.Component {
   render () {
     var article = this.props.article;
 
-    // TODO: ensure ArticlePreview can handle multiple authors
-    // TODO: edit link routing and remove /0/
+    // Render all contributing authors
+    function renderAuthors() {
```

Comment:
I don't think you can render undefined (in case there are no authors)?

Try returning null or an empty array instead.